styleguide/components/21-atoms/field-message/_field-message.scss

```diff
@@ -0,0 +1,75 @@
+.field-message {
+  @include theme('color', 'color-tertiary--lighten-1', 'field-messages-color');
+  @include icon('accolade-full', 'after');
```

Comment:
👎

Reply:
We'll need to discuss this one further next week.

Reply:
Decided to implement this through an SVG image for most instances. Pictures will need to be implemented with an inverted SVG.

Reply:
So we can remove the icon?

Reply:
I removed it for now, will implement the right one when I get it from one of the designers.